app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
You wouldn't include HTML in the `__()` function itself, only a placeholder (`{date}`). Then the `format()` function will replace `{date}` by the corresponding HTML. 

It's the same mechanism as `{listAuthor}`, which is replaced by the HTML defined in the `$link` variable.